docs/07-Debugging.md

```diff
@@ -1,45 +1,46 @@
 You can find the output of the factory by getting
 - `$index->getQueryFactory()`. See the [API documentation](https://firesphere.github.io/solr-api/classes/Firesphere.SolrSearch.Factories.QueryComponentFactory.html) on how to address each part of the factory.
 
-Through the use of an `Extension` on `BaseIndex` you can get the Factory through the method `onBeforeSearch`
+Through the use of an `Extension` on `BaseIndex`, you can get the Factory through the method `onBeforeSearch`
 
 These two methods supply more information about the executed query.
 
-For a more thorough debugging, have a look at the [Solarium docs](https://solarium.readthedocs.io/en/stable/queries/select-query/building-a-select-query/components/debug-component/) for getting the debugging information.
+For more thorough debugging, have a look at the [Solarium docs](https://solarium.readthedocs.io/en/stable/queries/select-query/building-a-select-query/components/debug-component/).
 
 # Logging
 
-Every error triggers the SolrLogger, to retrieve the errors from Solr and store them in the database.
+Every error triggers the `SolrLogger`, which retrieves the errors from Solr and stores them in the database.
 
-These error logs are deletable only in dev mode or by administrators.
+These error logs can only be deleted in dev mode or by administrators.
 
-The logs can be found at `admin/searchadmin/Firesphere-SolrSearch-Models-SolrLog` for checking what is wrong.
+The logs can be found at the URL `/admin/searchadmin/Firesphere-SolrSearch-Models-SolrLog`
 
 ## Clearing out the logs
 
-As an admin, you should have access to `dev/tasks`, where an option to truncate the log database is available at
-`dev/tasks/SolrClearErrorsTask`. Use this with caution though, as it will truncate the errors logged and no data remains
-at all.
+As an admin, you can truncate the log database via the dev task `dev/tasks/SolrClearErrorsTask`.
+
+Use this with caution though, as it will completely wipe the errors logged and no data will remain at all.
 
 It is strongly advised to only clear out the logs if they have all been reviewed and you are sure nothing serious is wrong.
 
 ## x:Unknown indexes
 
-Because Solr 5 and lower don't return the actual core that threw the error, logging will say `x:Unknown`. This does not mean
-the log is not there, it means the core that threw the error isn't known by the server.
+Up until Solr 5, the actual core that threw the error wasn't included in the error message. Logging will say
+`x:Unknown` in this circumstance. This does not mean the log is not there, it means the core that
+threw the error isn't known by the server.
 
 ## Sorting
 
-By default, SilverStripe sorts by ID, causing the oldest errors to show first. Advised is to sort by Timestamp before looking at the actual error.
+By default, SilverStripe sorts by ID, causing the oldest errors to show first - sorting by Timestamp is the simplest
```

Comment:
I think this part should be deleted, as we've added the default sort parameter already. But it's worth testing

Reply:
Happy to delete - should be fairly intuitive anyway I think